recipes/impactutils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.7" %}
+
+package:
+  name: impactutils
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/earthquake-impact-utils/archive/{{ version }}.tar.gz
+  sha256: 896081c5e7528c0d5ab11415dcd52037bcc0f5af0756956fa3ef74f68a2b34e1
+
+build:
+  number: 0
+  skip: True  # [not py35]
```

Comment:
Does this package not work on python 3.6?

Reply:
@synapticarbors It should, I didn't pay attention to what that "not py35" directive meant (dangers of cutting and pasting.) . I'll remove that.